Project3.md

```diff
@@ -329,25 +330,32 @@ mess. I recommend you do the following:
 
 ### Calling HTTP/JSON APIs in Python ###
 
-    import urllib.request
-    import urllib.parse
-    import json
-
-    # make a GET request and parse the returned JSON                                                                                                                                                           # note, no timeouts, error handling or all the other things needed to do this for real                                                                                                                      
-    print ("About to do the GET...")
-    req = urllib.request.Request('http://jsonplaceholder.typicode.com/posts/1')
-    resp_json = urllib.request.urlopen(req).read().decode('utf-8')
-    resp = json.loads(resp_json)
-    print(resp)
-
-    # make a POST request.                                                                                                                                                                                     # we urlencode the dictionary of values we're passing up and then make the POST request                                                                                                                    # again, no error handling                                                                                                                                                                                  
-    print ("About to do the POST...")
-    post_data = {'title': 'Demo Post', 'body': 'This is a test', 'userId': 1}
-    post_encoded = urllib.parse.urlencode(post_data).encode('utf-8')
-    req = urllib.request.Request('http://jsonplaceholder.typicode.com/posts', data=post_encoded, method='POST')
-    resp_json = urllib.request.urlopen(req).read().decode('utf-8')
-    resp = json.loads(resp_json)
-    print(resp)
+Python requests is a HTTP library with the mission of "HTTP for Humans". It is easier to use than the 
+standard python urllib/urllib2 packages. There is no need to encode/decode HTTP content, explicitly deserialize json, etc.
+Everything just "works" elegantly! However, since it is not a python 
```

Comment:
Is this a good thing or a bad thing for the purposes of teaching students about web services? Perhaps it's good to make them go through all the gory details in this class so they understand things more deeply? I'm not sure what the right answer is.